pkg/operation/seed/seed.go

```diff
@@ -323,6 +331,20 @@ func BootstrapCluster(seed *Seed, secrets map[string]*corev1.Secret, imageVector
 		sgFluentdCredentials := deployedSecretsMap["fluentd-es-sg-credentials"]
 		sgFluentdPassword = string(sgFluentdCredentials.Data[utilsecrets.DataKeyPassword])
 		sgFluentdPasswordHash = string(sgFluentdCredentials.Data[utilsecrets.DataKeyPasswordBcryptHash])
+
+		//Read extension provider specific configuration
+		existingConfigMaps := &corev1.ConfigMapList{}
+		if err = k8sSeedClient.Client().List(context.TODO(), existingConfigMaps, client.MatchingLabels(map[string]string{
```

Comment:
I guess we can add also `-n garden`.
```suggestion
		if err = k8sSeedClient.Client().List(context.TODO(), existingConfigMaps,
			client.InNamespace(common.GardenNamespace),
			client.MatchingLabels(map[string]string{v1alpha1.LabelExtensionConfiguration: v1alpha1.LabelLogging})); err != nil {
			return err
		}
```